src/main/java/com/cloudbees/jenkins/support/impl/GCLogs.java

```diff
@@ -147,6 +147,10 @@ private String getGcLogFileLocation() {
         }
         return gcLogSwitch.substring(GCLOGS_JRE_SWITCH.length());
     }
+    
+    public boolean isFileLocationParameterized(String fileLocation) {
+        return Pattern.compile(".*%[tnp].*").matcher(fileLocation).find();
```

Comment:
Is it possible a \n in the file location?

Reply:
It's %n, not resolved to \n I guess, but I couldn't find it documented. Just %t and %p

Reply:
`%n` was actually a feature request. But you are right it is not actually supported.